architecture/service_catalog/service_catalog_cli.adoc

```diff
@@ -0,0 +1,435 @@
++
+[source, bash]
+----
+$ svcat provision postgresql-instance --class rh-postgresql-apb --plan dev --params-json  '{"postgresql_database":"admin","postgresql_password":"admin","postgresql_user":"admin","postgresql_version":"9.6"}' -n szh-project
+  Name:        postgresql-instance
+  Namespace:   szh-project
+  Status:
+  Class:       rh-postgresql-apb
+  Plan:        dev
+
+Parameters:
+  postgresql_database: admin
+  postgresql_password: admin
+  postgresql_user: admin
+  postgresql_version: "9.6"
+----
+
+[[service-catalog-cli-view-serviceinstance]]
+===== View service instance details
+To view service instance details:
+[source, bash]
+----
+$ svcat get instance
+         NAME            NAMESPACE          CLASS         PLAN   STATUS
++---------------------+-------------+-------------------+------+--------+
+  postgresql-instance   szh-project   rh-postgresql-apb   dev    Ready
+----
+
+[[service-catalog-cli-create-servicebinding]]
+==== Create ServiceBinding
+
+When you create a `ServiceBinding` resource:
+
+. The service catalog controller communicates with the broker server to initiate
+the binding.
+. The broker server create credentials and issue them to the service catalog
+controller.
+. The service catalog controller adds those credentials as secrets to the
+project.
+
+Create the service binding using the command:
+
+[source, bash]
+----
+$ svcat bind postgresql-instance --name mediawiki-postgresql-binding
+  Name:        mediawiki-postgresql-binding
+  Namespace:   szh-project
+  Status:
+  Instance:    postgresql-instance
+
+Parameters:
+  {}
+----
+
+[[service-catalog-cli-view-servicebinding]]
+===== View service binding details
+. To view service binding details:
++
+[source, bash]
+----
+$ svcat get bindings
+              NAME                NAMESPACE         INSTANCE         STATUS
++------------------------------+-------------+---------------------+--------+
+  mediawiki-postgresql-binding   szh-project   postgresql-instance   Ready
+----
+
+. Verify the instance details after binding the service:
++
+[source, bash]
+----
+$ svcat describe instance postgresql-instance
+  Name:        postgresql-instance
+  Namespace:   szh-project
+  Status:      Ready - The instance was provisioned successfully @ 2018-06-05 08:42:55 +0000 UTC
+  Class:       rh-postgresql-apb
+  Plan:        dev
+
+Parameters:
+  postgresql_database: admin
+  postgresql_password: admin
+  postgresql_user: admin
+  postgresql_version: "9.6"
+
+Bindings:
+              NAME               STATUS
++------------------------------+--------+
+  mediawiki-postgresql-binding   Ready
+----
+
+
+[[service-catalog-cli-deleting-resources]]
+== Deleting resources
+To delete service catalog related resources, you need to unbind and deprovision
+the service instances.
+
+[[service-catalog-cli-delete-servicebindings]]
+=== Deleting service bindings
+
+. Unbind the service instance:
++
+[source, bash]
+----
+$ svcat unbind -n <project-name> <instance-name>
```

Comment:
This part doesn't seem to flow well to me. Since 1 and 2 are really the same step, I think they should be combined.

Also, it should be made clear that that when bindings are removed this way, the user doesn't have to explicitly name each binding to remove. But then give an example of how that is done, such as with `svcat unbind -n szh-project --name mediawiki-postgresql-binding`.